File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for i,j,k in commands:
        # i,j,k =idx
        # i, j, k = idx[0],idx[1],idx[2]
        new_array = []
        new_array= array[i-1:j]
        new_array.sort()
        answer.append(new_array[k-1])
        
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    const answer = [];
    for (cmd of commands) {
        const [i,j,k] = cmd;
        answer.push(array.slice(i-1,j).sort((a,b)=> (a-b))[k-1]);
    }
    return answer;
}